h2o-r/tests/testdir_algos/glm/runit_GLM_negbinomial_dispersion.R

```diff
@@ -0,0 +1,259 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
```

Comment:
**Warning:** `theta` in `MASS::glm.nb` and `MASS::theta.ml` corresponds to `1/theta` in h2o.